theme/tripal_analysis_blast.theme.inc

```diff
@@ -12,6 +12,18 @@ function tripal_analysis_blast_preprocess_tripal_feature_blast_results(&$variabl
     $feature->tripal_analysis_blast = new stdClass;
     $feature->tripal_analysis_blast->blast_results_list = $blast_results;
 
+    drupal_add_css(drupal_get_path('module', 'tripal_analysis_blast') . "/theme/css/tripal_analysis_blast.css");
+    drupal_add_js(drupal_get_path('module', 'tripal_analysis_blast') . "/theme/js/tripal_analysis_blast.js");
+
+    drupal_add_js("https://cdn.rawgit.com/calipho-sib/feature-viewer/v1.0.0/dist/feature-viewer.bundle.js", [
```

Comment:
Would it be possible to update to 1.0.4 as in https://github.com/tripal/tripal_analysis_interpro/issues/24 to avoid downloading 2 different version (one begin overridden by the other)?

Reply:
(and we'd better use the min version too)

Reply:
i think we need the full version unless we also include bootstrap?  Or perhaps thats no longer true now that we don't use the accordion.

Reply:
Ah nevermind, I hadn't seen it

Reply:
updated to 1.0.4